boa/contracts/vyper/vyper_contract.py

```diff
@@ -651,7 +651,11 @@ def debug_frame(self, computation=None):
             # TODO: figure out why fn is None.
             return None
 
-        frame_info = self.compiler_data.function_signatures[fn.name]._ir_info.frame_info
+        signatures = self.compiler_data.function_signatures
+        if fn.name not in signatures:
```

Comment:
@charles-cooper is there an easy way to get the signatures of imported modules or to get the errored module from the ast node?